### DISCUSSION TITLE
Paths doesn't work for me

### DISCUSSION THREAD

**Discussion by fawazahmed0** [community/community#26007] [Actions]
<p>Here is <a href="https://github.com/fawazahmed0/justatest/actions/runs/244762620/workflow" rel="nofollow noopener">workflow file</a> and <a href="https://github.com/fawazahmed0/justatest/actions/runs/244762620" rel="nofollow noopener">log</a></p>
<p>Here is <a href="https://github.com/fawazahmed0/justatest" rel="nofollow noopener">repo</a></p>
<p>Here is <a href="https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#filtering-for-specific-branches-tags-and-paths" rel="nofollow noopener">doc1</a> and <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths" rel="nofollow noopener">doc2</a> which I referred</p>
<p>I am not sure what I am doing wrong in here</p>

**Comment by kingthorin** [comment, marked as answer]
<p>You’ve just got things in the wrong order:</p>
<pre><code class="lang-auto">on:
  push:
    branches: [ master ]
    paths:
    - 'qdata/*'
  pull_request:
    branches: [ master ]
    paths:
    - 'qdata/*'
</code></pre>

**Comment by fawazahmed0** [comment]
<p>Thanks, I was struggling to understand, why the workflow was failing</p>
